carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -1540,8 +1540,26 @@ void WMBroadcaster::addGeofence(std::shared_ptr<Geofence> gf_ptr)
     if (update->affected_parts_.empty())
       continue;
 
-    // Process the geofence object to populate update remove lists
-    addGeofenceHelper(update);
+    // Process the geofence object to populate update/remove lists
+    try {
+      addGeofenceHelper(update);
+    }
+    catch (const lanelet::InvalidInputError& e) {
+      RCLCPP_WARN_STREAM(rclcpp::get_logger("carma_wm_ctrl"), 
+        "carma_wm_ctrl detected a potential issue in processing incoming MAP or Geofence update: " << e.what());
+
+      if (!j2735_map_msg_marker_array_.markers.empty()) {
+        RCLCPP_WARN_STREAM(rclcpp::get_logger("carma_wm_ctrl"), 
+          "Detected an attempt to add J2735 MAP msg. May not be error. Please verify J2735 MAP msg visualization or logs for more clues. "
```

Comment:
Can we update this log a little. If its throwing an exception i feel like its clearly an error - Maybe we remove the "May not be error" part and update the first line to something like "Encountered error trying to add j2735 MAP msg"

Reply:
It could be NOT error due to MAP msg data for undesired intersection. However, I am going to add that reason why.

Reply:
Moved the PR to https://github.com/usdot-fhwa-stol/carma-platform/pull/2508